app/script/calling/CallingRepository.js

```diff
@@ -1419,18 +1427,9 @@ z.calling.CallingRepository = class CallingRepository {
    * @returns {undefined} No return value
    */
   setDebugState(isDebuggingEnabled) {
-    const isStateChange = this.debugEnabled !== isDebuggingEnabled;
-    if (isStateChange) {
```

Comment:
Please keep the isStateChange check to skip obsolete log messages

Reply:
What do you mean by obsolete log messages? setDebugState is always on true now since we need to activate WebRTC adapter logging: https://github.com/wireapp/wire-webapp/pull/2797/files/61a6dca5039c627341dfad0de075c228f06acd8a#diff-584962c1c38d95a8eccb8fe4948ed7b8R106

Reply:
I see your point. But then you should rename the function and not use a parameter. Currently this is a feature to set the debug state. But now you always want to set it to true when initializing. So it turns into a private `_enableDebugging()` with no need to contain a log message at all.